gmail-to-salesforce-lead/README.md

```diff
@@ -0,0 +1,80 @@
+# Example to create leads obtained via email on Salesforce using the OpenAI API.
```

Comment:
Follow the pattern in existing samples. It should be something like
`Email lead info into Salesforce using OpenAI` Refer to https://github.com/ballerina-guides/integration-samples/blob/main/gcalendar-new-event-to-trello-card-and-twillio-msg/README.md etc

Reply:
Refer to the same link above and add single sentence with purpose.

Reply:
Fixed in [a718fba](https://github.com/ballerina-guides/integration-samples/pull/32/commits/a718fba876d7ed04e212bd0cd384c113178380cb)